ietf/templates/meeting/add_session_recordings.html

```diff
@@ -0,0 +1,113 @@
+{% extends "base.html" %}
+{# Copyright The IETF Trust 2015, All Rights Reserved #}
+{% load origin static django_bootstrap5 %}
+{% block title %}Add I-Ds to {{ session.meeting }} : {{ session.group.acronym }}{% endblock %}
+{% block pagehead %}{{ form.media.css }}{% endblock %}
+{% block content %}
+    {% origin %}
+    <h1>
+        Add Recordings to  {{ session.meeting }}
+        {% if session_number %}: Session {{ session_number }}{% endif %}
+        <br>
+        <small class="text-body-secondary">{{ session.group.acronym }}
+            {% if session.name %}: {{ session.name }}{% endif %}
+        </small>
+    </h1>
+    {% comment %} TODO: put the session name here or calculate the number at the meeting {% endcomment %}
+    {% if session.is_material_submission_cutoff %}
+        <div class="alert alert-warning my-3">
+            The deadline for submission corrections has passed. This may affect published proceedings.
+        </div>
+    {% endif %}
+    <form method="post" id="delete_recordings_form">
+        {% csrf_token %}
+        <h2 class="mt-5">Recordings already linked to this session</h2>
+        <table class="table table-sm table-striped tablesorter">
+            <thead>
+                <tr>
+                    <th scope="col" data-sort="num">Revision</th>
+                    <th scope="col" data-sort="document">Document</th>
+                    <th scope="col">Delete</th>
+                </tr>
+            </thead>
+            {% if already_linked %}
+                <tbody>
+                    {% for sp in already_linked %}
+                        <tr>
+                            <td>
+                                {% if sp.rev %}
+                                    -{{ sp.rev }}
+                                {% else %}
+                                    (current)
+                                {% endif %}
+                            </td>
+                            <td><a href="{{ sp.external_url }}">{{ sp.title }}</a></td>
+                            <td><button type="submit" aria-label="Delete {{ sp.title }}" class="btn btn-danger" name="delete" value="{{ sp.pk }}">Delete</button></td>
+                        </tr>
+                    {% endfor %}
+                </tbody>
+            {% else %}
+                <tbody>
+                    <tr>
+                        <td colspan="3" class="text-center font-italic">(none)</td>
+                    </tr>
+                </tbody>
+            {% endif %}
+        </table>
+    </form>
+    <dialog id="delete_confirm_dialog">
+        <p>Really delete <a href="#" id="delete_confirm_link"></a>?</p>
```

Comment:
Could we say `Really delete the link to` instead? We don't want to leave anyone with the impression that this will remove the recording at youtube.

Reply:
Ok, it looks like this now

![Screenshot from 2024-11-20 07-33-26](https://github.com/user-attachments/assets/219b9a0d-3e49-4f25-8b98-99ba14ac50d3)